protocols/identify/CHANGELOG.md

```diff
@@ -4,6 +4,14 @@
 
 - Update to `libp2p-core` `v0.36.0`.
 
+- Rename types as per [discussion 2174].
+  `Identify` has been renamed to `Behaviour`.
+  The `Identify` prefix has been removed from various types like `IdentifyEvent`.
+  Users should prefer importing the identify protocol as a module (`use libp2p::identify;`),
+  and refer to its types via `identify::`. For example: `identify::Behaviour` or `identify::Event`.
+
+  [discussion 2174]: https://github.com/libp2p/rust-libp2p/discussions/2174
```

Comment:
When the latest version does _not_ have an `unreleased` tag next to its version, it means that this version is already released and you need to add a new entry with the appropriate version bump, plus pull that version bump through the entire workspace, adding a changelog entry to each crate that depends on `libp2p-identify`.

Reply:
thanks for the explanation Thomas. I take it this should be a `minor` bump then right? Bumped to `0.40`